### ISSUE TITLE
Verify format of passed-along things from zephir from ingest

### ISSUE THREAD

**Issue by aelkiss** [hathitrust/post_zephir_processing#53]
This is for both `groove_full.tsv.gz` and `zephir_ingested_items.tsv.gz`.

```
aeu.ark:/13960/t30305j3b        AEU     AEU     ia      ia.cihm_76481
```

I think that's an HTID, a campus(?) id, a collection id, a digitization source, and an optional IA ID, but we should double-check that.

**Comment by aelkiss** [comment]
Clarifications:

`groove_full.tsv.gz` comes from Zephir (it is `groove_export_${zephir_date}.tsv.gz`).
`zephir_ingested_items.tsv.gz` *is* generated by post-zephir processing as a result of processing the incoming json from Zephir, so we **should** verify the format there.

**Comment by cscollett** [comment]
groove full is everything we have without an indication it's ingested yet. groove incremental is new things that are not ingested yet. The fields are: htid, source, collection code, digitizing source, and the AI scan id.

**Comment by aelkiss** [comment]
Thanks -- I will have to look at what's going on with `groove_full.tsv.gz` because the one I checked only has 6,777,967 lines, which makes me think it's only the non-ingested stuff. There might be some other processing going on on our end.

**Comment by cscollett** [comment]
Yes, that's right. We have a field where we record when we get notice an item is in the volume repository. But some people give us their whole catalog, with items that will never be scanned, so it's not surprising there are so many records.

**Comment by aelkiss** [comment]
Oh, I misread your comment -- you had said "everything we have without an indication it's ingested yet" but I think I inserted an extra "if", i.e. read it as everything you have regardless of whether or not it's ingested. I think we're all good then.
